scipy-1.0/paper.tex

```diff
@@ -218,6 +222,55 @@ \section*{Discussion}
 \textbf{Future development}.
 \textit{This section should include key issues: sparse arrays, ndimage pixel vs point, splines, fftpack vs. np.fft and linalg vs. np.linalg, under-maintained submodules.}
 
+\subsection{ndimage pixel vs point}
+
+\begin{itemize}
+  \item A pixel is not a little square
+    (http://alvyray.com/Memos/CG/Microsoft/6\_pixel.pdf)
+  \item Match edges vs match centers
+    (http://entropymine.com/imageworsener/matching/)
+\end{itemize}
+
+There are two competing models for representing image measurements:
+the first considers an image as a collection of pixels, square blocks
+of size 1x1 with centers offset by (0.5, 0.5).  This model corresponds
+closely with a camera sensor, that integrates light as it falls on
+(more-or-less) square elements, producing a digital readout for each.
+
+Another view is that of an image as a grid of point measurements.
+This model explicitly reject two notions encouraged by the pixel
+model: that our signal is best represented as a step-wise functio, and
+that a single measurement represents knowledge of the surrounding
+square area element.  Instead, we think of the image as a smoothly
+varying function, of which we have measurements at grid positions.
+
+While the representations may not seem very different, they have
+significant implications for image processing algorithms such as
+rescaling.
+
+{\tt ndimage} currently follows the pixel model, but we think that the
```

Comment:
does it consistently follow the pixel model? I thought it was mostly that, but not everywhere